code/handlers/trackservers.q

```diff
@@ -330,15 +330,21 @@ startup:{
     retry[]}
 
 // Check if required processes all connected
-reqprocsnotconn:{[requiredprocs] 
-    not all requiredprocs in exec proctype from .servers.SERVERS where .dotz.liveh[w]
+reqprocsnotconn:{[requiredprocs;typeorname]
+    not all requiredprocs in ?[`.servers.SERVERS;enlist (`.dotz.liveh;`w);();typeorname]
```

Comment:
What is this ?[] statement the parse of? I was thinking it was 

parse"exec proctype from .servers.SERVERS where .dotz.liveh w" 

but it's a bit different to that output

can you add a comment to say what that line is a parse of

Reply:
it's parse"exec typeorname from .servers.SERVERS where .dotz.liveh w", where typeorname is either proctype or procname
would it have been better to keep the name reqprocsnotconn for the function that only checks proctypes to avoid confusion? it's now called reqproctypesnotconn

Reply:
No I think that's fine